CHANGELOG.md

```diff
@@ -1,12 +1,69 @@
+   implement the `lightning_block_sync::poll::Poll` trait yourself (#956).
+ * `lightning::ln::peer_handler::PeerManager` no longer requires that no calls
+   are made to referencing the same `SocketDescriptor` after
+   `disconnect_socket` returns. This makes the API significantly less
+   deadlock-prone and simplifies `SocketDescriptor` implementations
+   significantly. The relevant changes have been made to `lightning_net_tokio`
+   and `PeerManager` documentation has been substantially rewritten (#957).
+ * `lightning::util::message_signing`'s `sign` and `verify` methods now take
+   secret and public keys by reference instead of value (#974).
+ * Substantially more information is now exposed about channels in
+   `ChannelDetails`. See documentation for more info (#984 and #988).
+ * Feerates charged when forwarding payments over channels is now set in
+   `ChannelConfig::fee_base_msat` when the channel is opened. For existing
+   channels, the value is set to the value provided in
+   `ChannelManagerReadArgs::default_config::channel_options` the first time the
+   `ChannelManager` is loaded in 0.0.99 (#975).
+ * We now reject HTLCs which are received to be forwarded out over private
+   channels unless `UserConfig::accept_forwards_to_priv_channels` is set. Note
+   that `UserConfig` is never serialized and must be provided via
+   `ChannelManagerReadArgs::default_config` at each start (#975).
+
+## Bug Fixes
+
+ * We now forward gossip messages to peers instead of only relaying
+   locally-generated gossip or sending gossip messages during initial sync
+   (#948).
+ * Correctly send `channel_update` messages to direct peers on private channels
+   (#949). Without this, a private node connected to an LDK node over a private
+   channel cannot receive funds as it does not know which fees the LDK node
+   will charge.
+ * `lightning::ln::channelmanager::ChannelManager` no longer expects to be
+   persisted spuriously after we receive a `channel_update` message about any
+   channel in the routing gossip (#972).
+ * Asynchronous `ChannelMonitor` updates (using the
+   `ChannelMonitorUpdateErr::TemporaryFailure` return variant) no longer cause
+   spurious HTLC forwarding failures (#954).
+ * Transaction provided via `ChannelMonitor::transactions_confirmed`
+   after `ChannelMonitor::best_block_updated` was called for a much later
+   block now trigger all relevant actions as of the later block. Previously
+   some transaction broadcasts or other responses required an additional
+   block be provided via `ChannelMonitor::best_block_updated` (#970).
+ * We no longer panic in rare cases when an invoice contained last-hop route
+   hints which were unusable (#958).
+
+## Node Compatibility
+
+ * We now accept spurious `funding_locked` messages sent prior to
+   `channel_reestablish` messages after reconnect. This is a
+   [known, long-standing bug in lnd] (https://github.com/lightningnetwork/lnd/issues/4006)
```

Comment:
I think you need to remove the space between the `]` and the `(`, otherwise the text is not linked:

https://github.com/rust-bitcoin/rust-lightning/blob/946ca58a59aed15169b22208fa7dc58f7cfc1cc2/CHANGELOG.md